tools/image_processing/imagej2/imagej2_noise/imagej2_noise.py

```diff
@@ -0,0 +1,100 @@
+# Galaxy wrapper for use with Imagej2 by Greg Von Kuster
+"""
```

Comment:
@bgruening I could give it a shot.  Do you want me to try to eliminate the imagej2_noise.py file altogether?   I assume I can make all of the same (or similar) calls from lines # 56 - 100 in Cheetah, but I'll need to see.